sw/airborne/firmwares/rotorcraft/guidance/guidance_h.c

```diff
@@ -555,6 +555,7 @@ void guidance_h_nav_enter(void)
   reset_guidance_reference_from_current_position();
 
   nav_heading = stateGetNedToBodyEulers_i()->psi;
+  guidance_h.sp.heading = stateGetNedToBodyEulers_f()->psi;
```

Comment:
This is an additional change and not only a result of changing it to float... what is the reason behind it?